app/components/windows/Informations.vue

```diff
@@ -35,6 +34,110 @@
 <style lang="less" scoped>
 @import "../../styles/index";
```

Comment:
見た感じmixinは未使用、色変数だけ使用していると思いますので、 #236 を受けてこのような変更を入れていただけると:pray: （ビルドが通れば良いのでCIの確認で十分です）

```suggestion
@import "../../styles/_colors";
```